tools/number-converter/src/NumberConverter.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+	<div class="NumberConverter">
+
+		<tool-markdown :markdown="`
+Enter a number to convert:
```

Comment:
I think we should add some info on how we do what we do here, similar to:
https://github.com/holistic-web/holistic-toolbox/blob/master/tools/json-formatter/src/JsonFormatter.vue#L5-L7

Reply:
8fe4db0